bin/proxy.js

```diff
@@ -0,0 +1,45 @@
+require('babel-register');
+
+const http = require('http');
+
+const bunyan = require('bunyan');
+const config = require('config');
+const httpProxy = require('http-proxy');
+
+const log = bunyan.createLogger({
+  name: 'proxy',
+  app: config.get('appName'),
+  serializers: bunyan.stdSerializers,
+});
+
+const proxy = httpProxy.createProxyServer();
+const apiHost = config.get('proxyApiHost', null) || config.get('apiHost');
+const frontendHost = `http://${config.get('serverHost')}:${config.get('serverPort')}`;
+
+log.info(`apiHost: ${apiHost}`);
+log.info(`frontendHost: ${frontendHost}`);
+
+function getHost(req) {
+  if (!req.headers['user-agent'].includes('Android') || req.url.startsWith('/api/')) {
```

Comment:
I updated it to just use the `mamo` cookie. So it will go to mobile unless the cookie is set to `off` or the path starts with `/api/`. This is probably a good enough start until work starts on the desktop frontend and we have a list of the URLs we want to allow setup somewhere.